src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -155,125 +151,116 @@ trait UserRoutes extends UserInfoDirectives with SamRequestContextDirectives {
                   }
                 }
             }
-          }
         }
       }
     }
 
-  def adminUserRoutes: server.Route =
+  def adminUserRoutes(samUser: SamUser, samRequestContext: SamRequestContext): server.Route =
     pathPrefix("admin") {
-      withSamRequestContext { samRequestContext =>
-        requireActiveUser(samRequestContext) { samUser =>
```

Comment:
Crazy question… What if `admin` apis actually didn't require the user to be an active user?  It's for admin things and we already need to be in `fc-admins` to do things.  In another conversation, we were already talking about frustration in needing to register our @firecloud.org emails in prod sam eventually in order to do some backend admin type things.  

It's off-topic question, but maybe something we can/should think about for performing admin things.  Might provide an easier way for bootstrapping a Sam instance so you don't have to directly insert a first user into the db…

Reply:
well, it is certainly more in the control fo sam code now. Registration never seemed like much of a burden to me. In any case, not this PR :)